vnext/Microsoft.ReactNative/Views/Image/Effects.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+
+#define EFFECTS_NAMESPACE winrt::Microsoft::ReactNative
+#include <BorderEffect.g.h>
+#include <GaussianBlurEffect.g.h>
+
+#pragma warning(push)
+#pragma warning(disable : 28285 28196 6387 6319 26812)
```

Comment:
I know this was here before, but it would be good if we could document why we're disabling warnings if we are.